scripts/flag_validator.bash

```diff
@@ -83,6 +90,11 @@ validate_existing_flags() {
 	find_global_flag_conflict
 
 	for cmd in "${swupd_commands[@]}"; do
+		# Commands which are not compiled dont have to be validated and
+		# can be skipped eg: 3rd-party
+		if check_command_exist "$cmd"; then
+			continue
+		fi
```

Comment:
this skips the 3rd-party commands when not installed, but when they are installed, how do you handle the extra argument in the command? I mean "3rd-party" is not really the command you  want to validate, but you want to validate `add`, `remove` and `list`